frontend/src/modules/search/components/FiltersPanel.js

```diff
@@ -33,23 +36,42 @@ type State = {|
 export class FiltersPanelBase extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
+
         this.state = {
             visible: false,
         };
     }
 
     toggleVisibility = () => {
-        this.setState((state) => {
+        this.setState(state => {
             return { visible: !state.visible };
         });
     }
 
-    selectStatus(status: string) {
-        const newStatus = status === 'all' ? null : status;
+    applyFilters = () => {
+        this.toggleVisibility();
+        return this.props.update();
+    }
+
+    resetFilters = () => {
+        this.props.resetStatuses();
+    }
+
+    createHandleSelectStatus = (status: string) => {
```

Comment:
I don't understand the name of this function, and the next one:
* why did you prepend names with `create`?
* why does this name include the word `handle, but the next one doesn't?

Reply:
They are called `createX` to make it obvious that they return a function, and don't perform any other actions themselves. The `handle` part I will remove.